src/main/java/com/segment/analytics/android/integrations/taplytics/TaplyticsIntegration.java

```diff
@@ -28,6 +28,16 @@
     }
   };
 
+  public static final Factory SKIPINIT = new Factory() {
```

Comment:
In other integrations, there only ever factory, hence it's a static member that's initialized when this class is loaded.

In this case, a customer would most likely use one factory but not both. By making this a static member, we're initializing the factory object twice (one for `SKIPINIT `and one for `FACTORY`).

Hence we should make instead this a function that returns a factory so that it's lazily initialized as and when a customer needs it.

```
public static Factory createSkipInitializationFactory() Factory {
  return new Factory...
}
```


This should also have JavaDocs explaining why a customer should use this.

I would also take a second look at the name. I don't have a great recommendation, but SKIPINIT sounds weird.